src/mgmt/cli_mgmt.c

```diff
@@ -697,6 +718,20 @@ dc_mgmt_net_cfg(const char *name, crt_init_options_t *crt_info)
 		"\tD_PROVIDER: %s, CRT_TIMEOUT: %d, CRT_SECONDARY_PROVIDER: %s\n",
 		crt_info->cio_provider, crt_info->cio_crt_timeout, buf);
 
+	/* Save attached ranks id info */
+	g_num_serv_ranks = resp->n_rank_uris;
+	serv_ranks_tmp   = NULL;
+	if (g_num_serv_ranks > 0) {
+		D_ALLOC_ARRAY(serv_ranks_tmp, g_num_serv_ranks);
+		if (serv_ranks_tmp == NULL)
+			D_GOTO(cleanup, rc = -DER_NOMEM);
+		for (idx = 0; idx < g_num_serv_ranks; idx++)
+			serv_ranks_tmp[idx] = resp->rank_uris[idx]->rank;
+	}
+	D_FREE(g_serv_ranks);
+	g_serv_ranks = serv_ranks_tmp;
+	D_INFO("Setting number of attached server ranks to %d\n", g_num_serv_ranks);
```

Comment:
would be good to combine this print with line 714 above

Reply:
> would be good to combine this print with line 714 above

- [ ] Merge status comment

Reply:
> > would be good to combine this print with line 714 above
> 
> * [x]  Merge status comment

Fixed with commit c8b0636